td-streamer-global.inc

```diff
@@ -286,6 +289,9 @@ stock Text:TDS_TextDrawCreate(Float:x, Float:y, const text[]) {
 
     data[E_TD_POS_X] = x;
     data[E_TD_POS_Y] = y;
+    
+    data[E_TD_EXTRA_1] = 0;
```

Comment:
0 is not a good default value for extra IDs as I've discovered with Incognito's streamer, I'd do something like this and then use it everywhere (and also add tests to check if the values are that by default):

`#define INVALID_TEXTDRAW_EXTRA_ID (cellmin)`